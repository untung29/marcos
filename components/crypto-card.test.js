import React from "react";
import renderer from "react-test-renderer";
// import { shallow } from "enzyme";
import CryptoCard from "./crypto-card";
import { withNavigation } from "react-navigation";

it("<CryptoCard/>", () => {
  const tree = renderer.create(withNavigation(<CryptoCard />)).toJSON();
  expect(tree).toMatchSnapshot();
});
