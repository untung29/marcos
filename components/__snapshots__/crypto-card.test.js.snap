// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CryptoCard/> 1`] = `null`;
